pytket/docs/index.rst

```diff
@@ -128,6 +128,7 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     
     Manual <https://cqcl.github.io/pytket/manual/index.html>
     Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
+    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples>
```

Comment:
is there an index.html file somewhere in the directory with more information?

Reply:
So I built the docs locally and then checked the built index.html file - the link seemed to work. Is this what you mean?

Reply:
I was thinking about changing the added link in a way that is point to an index.html file in the directory of the examples

Reply:
(see the comment below)